lib/active_merchant/billing/gateways/shift_four.rb

```diff
@@ -1,15 +1,15 @@
 module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
-    class Shift4Gateway < Gateway
+    class ShiftFourGateway < Gateway
```

Comment:
That should not be the problematic i think but since we are changing the identifier to `shift_four` and filenames are changed as well so I thought class name should be in sync with the filename?

Reply:
Class names and filenames should be the same. After looking, I think we should keep this as `Shift4` instead of renaming it too `ShiftFour`. There are other gateways with integer values in the class name and it's not a problem so long as it's not the leading character.

Reply:
Sure, updated !